core/src/main/java/org/infinispan/protostream/impl/TagWriterImpl.java

```diff
@@ -309,6 +309,8 @@ public ProtoStreamWriterImpl getProtoStreamWriter() {
 
    //todo [anistor] need to provide a safety mechanism to limit message size in bytes and message nesting depth on write ops
    private abstract static class Encoder {
+
+      protected final StringUtil.Utf8Helper utf8Helper = StringUtil.UTF8_HELPER;
```

Comment:
```suggestion
      protected static final StringUtil.Utf8Helper utf8Helper = StringUtil.UTF8_HELPER;
```

Reply:
Thanks, I thought I had done this! :see_no_evil: